### DISCUSSION TITLE
How to tell scala-cli to download & provide a library JAR?

### DISCUSSION THREAD

**Discussion by halloleo** [VirtusLab/scala-cli#2904] [General]
Newbie question: If I want to use an external library like fast parse in Scala, I write in the source file,

```
import fastparse.all._
.
.
.
```

But how do I tell scala-cli to download & provide the JAR for it?

    $ scala-cli <path/to/my/source-file>

gives me the error "Not found: fastparse".

**Comment by tgodzik** [comment]
Take a look at our docs, there is plenty of examples there https://scala-cli.virtuslab.org/docs/guides/introduction/configuration

**Reply by halloleo** [reply]
Thank you. I saw the `using` directive in blog posts, but didn't understand that it needs to be a *comment*...
